packages/frint-react/package.json

```diff
@@ -38,7 +38,9 @@
     "prop-types": "^15.5.10",
     "react": "^15.5.4",
     "react-dom": "^15.5.4",
-    "react-test-renderer": "^15.5.4"
+    "react-test-renderer": "^15.5.4",
```

Comment:
because `enzyme`, `prop-types`, `react`, `react-dom` , `react-test-renderer` are now in root `package.json`, they can be removed this `package.json`